source/kraken/metrics.h

```diff
@@ -81,6 +84,8 @@ class Metrics : boost::noncopyable {
     void observe_data_loading(double duration) const;
     void observe_data_cloning(double duration) const;
     void observe_handle_rt(double duration) const;
+    void observe_handle_disruption(double duration) const;
```

Comment:
can you name it `observe_handle_chaos`? Among all theses names, I was quite lost